tests/rbd_mirror/test_rbd_mirror.py

```diff
@@ -54,7 +54,14 @@ def run(**kw):
         mirror1.wait_for_status(imagespec=imagespec, state_pattern="up+stopped")
         mirror2.wait_for_status(imagespec=imagespec, state_pattern="up+replaying")
         mirror1.check_data(peercluster=mirror2, imagespec=imagespec)
-        mirror1.clean_up(peercluster=mirror2, pools=[poolname])
+
+        # Attempt creating mirror in same cluster this should be prevented
```

Comment:
test_rbd_mirror.py is being used as gateway for rbd-mirorr suites
I think it doesn't make sense to add a negative scenario atleast in this module.

Reply:
ok got your point, what is the suggestion to avoid multiple polarion ID?
Is that new module itself?

Reply:
For now it appears like yes
if you can comeup with any other solution, that would be interesting